scripts/unittest.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+# TODO : Need optimization. Can be exec without bootstrap I think
```

Comment:
Open to talk about that 😉

Reply:
I wanted to create a script to remember the command to execute, but I think it is possible to execute the tests with simpler commands